### DISCUSSION TITLE
Possible to buy this controller?

### DISCUSSION THREAD

**Discussion by mundsen** [Expatria-Technologies/grblhal_2000_PrintNC#2] [General]
If so, where?

**Comment by Cncninja** [comment]
same?

**Reply by andrewmarles** [reply]
Sorry for the late replys, I was not getting notifications on these discussion items (no @andrewmarles)

At this time, these boards are not available for sale commercially.

**Reply by andrewmarles** [reply]
Out of interest, if anyone is looking to source one of these boards, the PrintNC community is quite active in supplying them and you can likely pick one up from one of the members there.

https://wiki.printnc.info/en/grbl/GRBLHAL2000-marketplace
